README.md

```diff
@@ -18,16 +18,16 @@ The project follows [semantic versioning](https://semver.org/). In particular, w
 
 The overall approach is for Terravalet to generate migration scripts, not to perform any change directly. This for two reasons:
 
-1. Safety. The operator can review the generated migration scripts for correctness.
+1. Safety. The operator can review the migration scripts for correctness.
 2. Gitops-style. The migration scripts are meant to be stored in git in the same branch (and thus same PR) that performs the Terraform changes and can optionally be hooked to an automatic deployment system.
 
-Terravalet takes as input the output of `terraform plan` per each involved root module and generates one UP and one DOWN migration script.
+Terravalet takes as input the output of `terraform plan` for each involved root module and generates one UP and one DOWN migration script.
 
 ### Remote and local state
 
-At least until Terraform 0.14, `terraform state mv` has a bug: if a remote backend for the state is configured (which will always be the case for prod), it will remove entries from the remote state but it will not add entries to it. It will fail silently and leave an empty backup file, so you loose your state.
+At least until Terraform 0.14, `terraform state mv` has a bug: if a remote backend for the state is configured (which will always be the case for prod), it will remove entries from the remote state but it will not add entries to it. It will fail silently and leave an empty backup file, so you will loose your state.
```

Comment:
Just for my info: I am wondering, is this still the case with the newer terraform versions?

Reply:
I was wondering the same. I think that something has changed, since now the `-state` and `-state-out` options are described as "legacy". I didn't investigate further.

My current, superficial reasoning is that, also if it is fixed, I am not sure we want to move directly the remote state...

I will create a separate PCI ticket to investigate in depth.

Reply:
Created ticket: "terravalet: investigate if terraform bug about state move has been fixed" PCI-2326